mnefun/_report.py

```diff
@@ -0,0 +1,364 @@
+                    projs = read_proj(op.join(p.work_dir, subj, p.pca_dir,
+                                              'preproc_blink-proj.fif'))
+                    figs.append(plot_projs_topomap(projs, info=sss_info,
+                                                   show=False))
+                if any(p.proj_nums[2]):  # ERM
+                    captions.append('%s: Continuous' % section)
+                    projs = read_proj(op.join(p.work_dir, subj, p.pca_dir,
+                                              'preproc_cont-proj.fif'))
+                    figs.append(plot_projs_topomap(projs, info=sss_info,
+                                                   show=False))
+                report.add_figs_to_section(figs, captions, section,
+                                           image_format='svg')
+                print('%5.1f sec' % ((time.time() - t0),))
+            else:
+                print('    %s skipped' % section)
+
+            #
+            # Source alignment
+            #
+            section = 'Source alignment'
+            if p.report_params.get('source_alignment', True) and has_sss:
+                assert sss_info is not None
+                t0 = time.time()
+                print(('    %s ... ' % section).ljust(ljust), end='')
+                captions = ['Left', 'Front', 'Right']
+                captions = ['%s: %s' % (section, c) for c in captions]
+                try:
+                    from mayavi import mlab
+                except ImportError:
+                    warnings.warn('Cannot plot alignment in Report, mayavi '
+                                  'could not be imported')
+                else:
+                    subjects_dir = mne.utils.get_subjects_dir(
+                        p.subjects_dir, raise_error=True)
+                    bem, src, trans, _ = _get_bem_src_trans(
+                        p, sss_info, subj, struc)
+                    offscreen = mlab.options.offscreen
+                    mlab.options.offscreen = True
+                    tempdir = mne.utils._TempDir()
+                    if len(mne.pick_types(sss_info)):
+                        coord_frame = 'meg'
+                    else:
+                        coord_frame = 'head'
+                    try:
+                        fig = mlab.figure(bgcolor=(0., 0., 0.),
+                                          size=(1000, 1000))
+                        kwargs = dict(
+                            info=sss_info, subjects_dir=subjects_dir, bem=bem,
+                            dig=True, coord_frame=coord_frame, show_axes=True,
+                            fig=fig, trans=trans, src=src)
+                        try:
+                            mne.viz.plot_alignment(surfaces='head-dense',
+                                                   **kwargs)
+                        except Exception:
+                            mne.viz.plot_alignment(surfaces='head', **kwargs)
```

Comment:
I am hitting this with data with `fsaverage` as surrogate src space

>   File "/home/ktavabi/Projects/mnefun/mnefun/_report.py", line 225, in gen_html_report
>     mne.viz.plot_alignment(surfaces='head', **kwargs)
>   File "<string>", line 2, in plot_alignment
>   File "/home/ktavabi/Projects/mne-python/mne/utils.py", line 728, in verbose
>     return function(*args, **kwargs)
>   File "/home/ktavabi/Projects/mne-python/mne/viz/_3d.py", line 786, in plot_alignment
>     bem, FIFF.FIFFV_BEM_SURF_ID_HEAD)
>   File "/home/ktavabi/Projects/mne-python/mne/bem.py", line 1452, in _bem_find_surface
>     % name.replace('_', ' '))
> RuntimeError: BEM model does not have the head triangulation

Reply:
Should be fixed now